src/components/Navbar/MenuItems.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
```

Comment:
There is no need to import the whole react library into the file from the newest version of it.